### ISSUE TITLE
DE Feature normalization

### ISSUE THREAD

**Issue by edugm94** [xueyunlong12589/DGCNN#6]
Hello, 

I would like to know if the features are normalised or scaled or had any other type of preprocessing. 
Thanks in advance!

**Comment by xueyunlong12589** [comment]
I added a BN layer to the first layer of the network, so there was no pre-processing of the DE feature. Of course, it is also possible to remove the BN layer and normalize or standardize the data before entering the network

**Comment by edugm94** [comment]
Alright, thanks. I am trying to replicate the results from the paper and I was not able to get the 79.95 +- 9.02 in the leave-one-subject-out experiment (Table 3 from the paper).

**Comment by edugm94** [comment]
Hello, can it be possible to get the hyperparameters used during training?

**Comment by xueyunlong12589** [comment]
This is the&nbsp;open-source code of the paper's author, you can check it.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xueyunlong12589/DGCNN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月24日(星期五) 凌晨0:37
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [xueyunlong12589/DGCNN] DE Feature normalization (Issue #6)





 
Hello, can it be possible to get the hyperparameters used during training?

**Comment by edugm94** [comment]
Thanks! I tried to access here: [http://aip.seu.edu.cn/songtengfei](http://aip.seu.edu.cn/songtengfei) but the website does not exist or is inaccessible. Please, if you could provide the training script it would be great! Thanks :)
